Programatic usage need more arguments then the example file.

`jsDoc` and `topRef` properties must be defined (they are not optional).

The Readme example seems to ignore it. When you run in inside a ts  file, it throw this type error:
```
error TS2739: Type '{ path: string; tsconfig: string; type: string; expose: "export"; }' is missing the following properties from type 'Config': topRef, jsDoc
```

--- a/README.md
+++ b/README.md
@@ -36,7 +36,9 @@ const config = {
     path: "path/to/source/file",
     tsconfig: "path/to/tsconfig.json",
     type: "*", // Or <type-name> if you want to generate schema for that one type only
-    expose: "export",
+    expose: "export", // Default when using cli
+    jsDoc: "extended", // Default when using cli
+    topRef: true, // Default when using cli
 };
 
 const output_path = "path/to/output/file";

--- a/README.md
+++ b/README.md
@@ -36,9 +36,9 @@ const config = {
     path: "path/to/source/file",
     tsconfig: "path/to/tsconfig.json",
     type: "*", // Or <type-name> if you want to generate schema for that one type only
-    expose: "export", // Default when using cli
-    jsDoc: "extended", // Default when using cli
-    topRef: true, // Default when using cli
+    expose: "export",
+    jsDoc: "extended",
+    topRef: true
 };
 
 const output_path = "path/to/output/file";

